### PR TITLE
GSYE-780: Added ability to read fees from the database. Added ability…

### PR DESCRIPTION
**Pull request by spyrostz**
… to disable the home self-consumption, in order to handle the case where we have meter profiles that do not include the behind-the-meter self consumption.

## Reason for the proposed changes

Please describe what we want to achieve and why.

## Proposed changes

-

**INTEGRATION_TESTS_BRANCH**=feature/GSYE-780-db-tests
**GSY_FRAMEWORK_BRANCH**=master

### CODE REVIEW

File: src/gsy_e/gsy_e_core/user_profile_handler.py [hunk 1]
```diff
@@ -385,12 +439,29 @@ def current_timestamp(self) -> DateTime:
     def _update_current_time(self, timestamp: DateTime):
         self._current_timestamp = timestamp
 
+    @property
+    def current_scm_profiles(self) -> Dict[str, SCMFeesRatesProfileDatapoint]:
```
Comment (hannesdiedrich) [inline src/gsy_e/gsy_e_core/user_profile_handler.py:443]: I do not see this being used in the calculation. Can you please point me to it?

File: src/gsy_e/models/area/scm_dataclasses.py [hunk 2]
```diff
@@ -261,6 +275,7 @@ class AreaFees:
     grid_export_fee_const: float = 0.0
     grid_fees_reduction: float = 0.0
     per_kWh_fees: Dict[str, FeeContainer] = field(default_factory=dict)
+    per_kW_fees: Dict[str, ContractedPowerFeeContainer] = field(default_factory=dict)
```
Comment (hannesdiedrich) [inline src/gsy_e/models/area/scm_dataclasses.py:278]: I think you need to add this to the `prices_as_dict` in order to add those fees to the bill.

File: src/gsy_e/gsy_e_core/simulation/simulation.py [hunk 3]
```diff
@@ -542,7 +541,10 @@ def _cycle_markets(self, slot_no: int) -> None:
             self._get_current_market_time_slot(slot_no), area=self.area
         )
 
-        self.area.cycle_coefficients_trading(self.progress_info.current_slot_time)
+        self.area.cycle_coefficients_trading(
+            self.progress_info.current_slot_time,
+            global_objects.profiles_handler.current_scm_profiles,
```
Comment (spyrostz) [inline src/gsy_e/gsy_e_core/simulation/simulation.py:546]: @hannesdiedrich this is where the fee profiles are passed to the areas. Related to this comment:
https://github.com/gridsingularity/gsy-e/pull/1803/files#r1802887433

File: src/gsy_e/models/area/scm_dataclasses.py (hunk 2 above)
Comment (spyrostz) [reply src/gsy_e/models/area/scm_dataclasses.py:278]: Indeed. This PR is following shortly

**Pull request merged**